support-files/CMakeLists.txt

```diff
@@ -1,15 +1,15 @@
 # Copyright (c) 2006, 2016, Oracle and/or its affiliates.
 # Copyright (c) 2012, 2017, MariaDB
-# 
+#
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; version 2 of the License.
-# 
+#
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
-# 
+#
 # You should have received a copy of the GNU General Public License
```

Comment:
I hit these also.. Please put spaces back to have more nicer patch. I think those space are worth of own PR

Reply:
I have now removed those trailing space cleanups from this PR.